.travis.yml

```diff
@@ -0,0 +1,13 @@
+language: go
+
+go:
+ - 1.8
+
+go_import_path: github.com/specialedge/hangar-api
+
+install:
+  - go get github.com/gorilla/mux
+  - go get github.com/cavaliercoder/grab
+
+script:
+ - go test -v ./...    
```

Comment:
I'd like to add golint to the Travis script just to add an extra level of consistency guarantee. Example of that here: https://gist.github.com/y0ssar1an/df2dab474520c4086926f672c52db139#file-travis-yml-L46